en/orm/table-objects.rst

```diff
@@ -294,7 +302,16 @@ beforeDelete
 .. php:method:: beforeDelete(Event $event, EntityInterface $entity, ArrayObject $options)
 
 The ``Model.beforeDelete`` event is fired before an entity is deleted. By
-stopping this event you will abort the delete operation.
+stopping this event you will abort the delete operation. When the event is stopped the result
+of the event will be returned.
+Stopping the event works like this::
```

Comment:
That could be an option, too. It would be this link, right? **https://book.cakephp.org/3.0/en/core-libraries/events.html#stopping-events**
I hope it's obvious how to call it than 😀